DigitalExcellence/dex-backend
# Feature/364 integration tests

## Description

This PR adds the foundation for the API intergration tests. This PR is replacing Postman for new intergration tests.
A few tests are already made to test the functionallity of this PR, you as a reviewer should make sure these tests pass and verify that the code quality is on point for this PR.


## Type of change


## Checklist


## Steps to Test or Reproduce
02_API.Tests is the new XUnit integration test project which will replace Postman.

1. Pull the branch
2. Open the test explorer
3. Run all tests or 02_API.Tests and all tests should pass.

## Link to issue

Closes: #364
